### PR TITLE
Feature/getHostSpecs

### PR DESCRIPTION
**Pull request by andreGarvin**
@rdepena @datamadic @whyn07m3

### CODE REVIEW

File: src/api/system/system.ts [hunk 1]
```diff
@@ -238,6 +238,15 @@ export default class System extends Base {
     }
 
     /**
+     * Retrieves system information.
+     * @tutorial system.getHostSpecs
+     * @return {Promise.<Object>}
```
Comment (whyn07m3) [inline src/api/system/system.ts]: \<Object\> should be typed

File: src/api/system/system.ts [hunk 2]
```diff
@@ -238,6 +238,15 @@ export default class System extends Base {
...
+     */
+    public getHostSpecs(): Promise<Object> {
```
Comment (whyn07m3) [inline src/api/system/system.ts]: \<Object\> should be typed

File: test/system.test.ts [hunk 3]
```diff
@@ -82,6 +82,11 @@ describe('System.', () => {
         it('Fulfilled', () => fin.System.getProxySettings().then(() => assert(true)));
     });
 
+    describe('getHostSpecs()', () => {
+
```
Comment (whyn07m3) [inline test/system.test.ts]: Extra empty line

File: test/system.test.ts [hunk 4]
```diff
@@ -82,6 +82,11 @@ describe('System.', () => {
...
+        it('Fulfilled', () => fin.System.getHostSpecs().then(specs => assert(true)));
```
Comment (whyn07m3) [inline test/system.test.ts]: • `specs` here is unused. Can simply forgo it: `.then(() => assert(true)));`
• Should there be a test checking what is returned and its shape?

File: tutorials/System.getHostSpecs.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+Retrieves system information.
+
+# Example
+```js
+ fin.System.getHostSpecs().then(specs => console.log(specs)).catch(err => console.log(err));
```
Comment (whyn07m3) [inline tutorials/System.getHostSpecs.md]: Can be shorter: `fin.System.getHostSpecs().then(console.log).catch(console.error);`

File: src/api/system/system.ts (hunk 1 above)
Comment (andreGarvin) [reply src/api/system/system.ts]: @rdepena What do you mean by object should be typed ?

**Pull request merged**